osf_tests/test_search_views.py

```diff
@@ -34,7 +34,7 @@ def tearDown(self):
         import website.search.search as search
         search.delete_all()
 
-    def test_search_contributor(self):
```

Comment:
Looks like you went juuuust a little too far. This is all in the tear down method. Maybe have `def test_search(self):` here.